Documentation/operations/upgrade.rst

```diff
@@ -341,6 +341,16 @@ Removed Metrics
 Changed Metrics
 ~~~~~~~~~~~~~~~
 
+The following metrics previously had instances (i.e. for some watcher K8s resource type labels) under `workqueue_`.
+In this release any such metrics have been renamed and combined into the correct metric name prefixed with `cilium_operator_`.
+
+* `workqueue_adds_total` has been renamed and combined into to `cilium_operator_workqueue_adds_total`.
```

Comment:
note: As of builds following commit 1a52e9f2509fbc3ced930fdcd34a44d055c7ec2c these are likely coming out under the cilium agent namespace (ex. `cilium_workqueue_adds_total`).  However, the last release would still be calling these just `workqueue_...`.